src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -321,6 +321,7 @@ void uct_rc_mlx5_release_desc(uct_recv_desc_t *self, void *desc);
 typedef struct uct_rc_mlx5_release_desc {
     uct_recv_desc_t             super;
     unsigned                    offset;
+    unsigned                    payload_offset;
```

Comment:
no need to save it since it's known in compile time
`sizeof(uct_iface_recv_desc_t)`